src/main/scala-2_11/scala/tools/refactoring/implementations/oimports/TreeToolboxScanners.scala

```diff
@@ -29,5 +29,10 @@ class TreeToolboxScanners[G <: Global](val global: G) {
     }
 
     def comments = comments_.toList
+
+    override def emitIdentifierDeprecationWarnings = false
+
+    override def error(off: Offset, msg: String): Unit =
+      if (name != global.nme.MACROkw) super.error(off, msg)
```

Comment:
for this particular scanner only. It's role is just to pick comments so `macro` keyword should not hold it